data/human/free worlds 1 start.txt

```diff
@@ -1034,7 +1034,7 @@ mission "FW Senate 1B"
 				`	"I have no idea."`
 				`	"Ummm... they both involve momentous decisions that have serious consequences?"`
 				`	"They both require someone skilled and decisive to be in command?"`
-			`	"The correct answer," she says, "was, 'nothing whatsoever; they are entirely different disciplines.' Now, hurry up, I hope to be on Bourne a week from now." You consider explaining to her that it's impossible to get there that fast, but decide against it.`
+			`	"The correct answer," she says, "was, 'nothing whatsoever; they are entirely different disciplines.' Now, hurry up, I hope to be on Bourne a week from now." You consider explaining to her that hyperlanes don't allow you to go that fast, but decide against it.`
 				accept
```

Comment:
```suggestion
			`	"The correct answer," she says, "was, 'nothing whatsoever; they are entirely different disciplines.' Now, hurry up, I hope to be on Bourne a week from now." You consider explaining to her that it's impossible to get there that fast, even with a hyperdrive, but decide against it.`
			branch jump
				has "outfit (flagship installed): Jump Drive"
				accept
			label jump
			`	It's a good thing you have a jump drive.`
				accept
```
This gives a bit more flavor based on whether the player has a jump drive. Thoughts?

Reply:
If I'm reading this correctly, then if you have a Jump Drive (I also sense a capitalization discussion), it would read:
			`	"The correct answer," she says, "was, 'nothing whatsoever; they are entirely different disciplines.' Now, hurry up, I hope to be on Bourne a week from now." You consider explaining to her that it's impossible to get there that fast, even with a hyperdrive, but decide against it.`
			`	It's a good thing you have a jump drive.`

I would say `using a hyperdrive`, rather than `even with a hyperdrive`.

Reply:
Looks like `jump drive` is most often/always lowercase when used in conversation (based on a couple text files I looked at), so I think we're good there.

```suggestion
			`	"The correct answer," she says, "was, 'nothing whatsoever; they are entirely different disciplines.' Now, hurry up, I hope to be on Bourne a week from now." You consider explaining to her that it's impossible to get there that fast using a hyperdrive, but decide against it.`
			branch accept
				not "outfit (flagship installed): Jump Drive"
			`	It's a good thing you have a jump drive.`
				accept
```